ansible/postdeploy.yml

```diff
@@ -6,4 +6,9 @@
 - hosts: ansible
   tasks:
     - include: tasks/installCatalog.yml
-      when: mode == "deploy"
\ No newline at end of file
+      when: mode == "deploy"
+
+- hosts: ansible
+  tasks:
+    - include: tasks/installOpenwhiskCatalog.yml
```

Comment:
@rabbah what do you mean by "list of GitHub repos"?
openwhisk-catalog is just one repo. I am not sure if I catch you. Could you elaborate more? Thank you.